### PR TITLE
Use terminal detach keys sequence specified in the config file

### PR DESCRIPTION
**Pull request by marcov**
Fixes: #4556

Signed-off-by: Marco Vedovati <mv@sba.lat>

### CODE REVIEW

**Comment by mheon** [comment]
Lint seems to be complaining about a bit of excess whitespace?

File: pkg/adapter/containers.go [hunk 1]
```diff
@@ -367,6 +367,23 @@ func (r *LocalRuntime) CreateContainer(ctx context.Context, c *cliconfig.CreateV
 	return ctr.ID(), nil
 }
 
+// Select the detach keys to use from user input flag, config file, or default value
+func (r *LocalRuntime) selectDetachKeys(flagValue string) (error, string) {
```
Comment (mheon) [inline pkg/adapter/containers.go]: Should probably flip this to (string, error)

File: cmd/podman/exec.go [hunk 2]
```diff
@@ -36,6 +36,9 @@ func init() {
 	flags := execCommand.Flags()
 	flags.SetInterspersed(false)
 	flags.StringVar(&execCommand.DetachKeys, "detach-keys", define.DefaultDetachKeys, "Select the key sequence for detaching a container. Format is a single character [a-Z] or ctrl-<value> where <value> is one of: a-z, @, ^, [, , or _")
+	// Clear the default, the value specified in the config file should have the
+	//priority
```
Comment (mheon) [inline cmd/podman/exec.go]: Missing a space between // and priority

**Comment by TomSweeneyRedHat** [comment]
Other than @mheon's comments, LGTM

**Comment by marcov** [comment]
Updated, thank you!

**Comment by rhatdan** [comment]
Thanks @marcov

**Pull request merged**